### ISSUE TITLE
changing bold font in lux theme

### ISSUE THREAD

**Issue by taylorreiter** [mkdocs/mkdocs-bootswatch#69]
Hello! I've been using the lux theme for my mkdocs site, however bold font isn't substantially different from standard text. I've been trying to find a way to edit my `css/extra.css` file to change this, but I haven't been able to figure out how to do this. Is there a way to adapt the lux theme this way?

**Comment by andburch** [comment]
+1

**Comment by andburch** [comment]
I don't think this is a GREAT solution, but if you can insert this CSS to override the default, it works. I'm using Quarto (not MKdocs), I think the problem is associated with using the "bolder" option for font weight. Changing it to "bold" makes normal bold font look better?
```
b,strong {font-weight: bold}
```

**Comment by t-emery** [comment]
+1
